package.json

```diff
@@ -28,11 +28,9 @@
     "@angular/platform-browser-dynamic": "~8.2.10",
     "@angular/router": "~8.2.10",
     "@types/file-saver": "^2.0.1",
-    "@ngx-translate/core": "^12.1.2",
-    "@ngx-translate/http-loader": "^4.0.0",
     "ajv": "^6.4.0",
     "ajv-keywords": "^3.4.0",
-    "arlas-wui-toolkit": "13.0.0",
+    "arlas-wui-toolkit": "14.0.0-beta.5",
     "arlas-persistence-api": "13.0.0-beta.2",
```

Comment:
we can remove arlas-persistence-api import ?

Reply:
yes totally